mlflow/sklearn-triton/wine/conda.yaml

```diff
@@ -0,0 +1,12 @@
+name: example
+channels:
+  - defaults
+dependencies:
+  - pip
+  - python=3.6
+  - numpy=1.14.3
+  - pandas=0.25.3
+  - scikit-learn=0.19.1
+  - joblib=1.0.0
+  - pip:
+    - mlflow
```

Comment:
I guess we should fix the `1.13` version of `mlflow` to avoid issues with `scipy`?

Reply:
Its training conda.yaml file
For serving needs conda.yaml is located in ./data folder and there is no mlflow dependency inside at all